lib/process-services-cloud/src/lib/process/process-list/components/process-list-cloud.component.ts

```diff
@@ -210,7 +222,17 @@ export class ProcessListCloudComponent extends DataTableSchema implements OnChan
     }
 
     ngAfterContentInit() {
-        this.createDatatableSchema();
+        this.isLoadingColumnsOrder = true;
+        this.loadColumnsOrderPreferences()
```

Comment:
loading column order from preferences service. Other solution would be to pass `columnOrder` from parent component (e.g. in alfresco-apps/process-list-cloud-ext.component) but we would need to delay displaying of the datatable body e.g. by passing flag in order to showing the loading spinner (loading spinner is disappear right after loading processes).